### PR TITLE
Create a System.Runtime.InteropServices.Analyzers project

### PR DESCRIPTION
**Pull request by srivatsn**
Three rules related to PInvokes are being moved here (CA1060, CA1401, CA2101).

The rules are fairly straightforward and dont branch out into C#\VB. The fixer for CA2101 needs to be able to fix DeclareSub statements in VB and needs language specific dlls.

### CODE REVIEW

**Comment by srivatsn** [comment]
@mavasani @shyamnamboodiripad @tmeschter can you please take a look?

File: src/Roslyn.sln [hunk 1]
```diff
@@ -331,6 +331,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "vbc", "Compilers\VisualBasi
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Pdb2Xml", "Tools\Source\Pdb2Xml\Pdb2Xml.csproj", "{CF450DCE-D12B-4A11-8D2D-A7A125372C48}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "System.Runtime.InteropServices.Analyzers", "System.Runtime.InteropServices.Analyzers", "{32BCFD38-B44B-4E4F-AFC6-90ABAF32C465}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SystemRuntimeInteropServicesAnalyzers", "Diagnostics\FxCop\System.Runtime.InteropServices.Analyzers\Core\SystemRuntimeInteropServicesAnalyzers.csproj", "{0A2BB2D6-883E-4C19-895A-0DC067436623}"
```
Comment (mavasani) [inline src/Roslyn.sln]: We don't need the VSIX setup project, just like for other group of analyzer projects?

Comment (srivatsn) [inline src/Roslyn.sln]: Hmm ya.. I hadn't created a VSIX project for System.Runtime.Analyzers either. I'll add them both in a later checkin.

**Comment by mavasani** [comment]
You may want to do a quick check that none of our tests actually rely on these diagnostics coming from their current analyzer projects.

:+1:

**Pull request merged**